clang/lib/StaticAnalyzer/Checkers/WebKit/RawPtrRefMemberChecker.cpp

```diff
@@ -134,10 +137,10 @@ class NoUncountedMemberChecker
     Os << " in ";
     printQuotedQualifiedName(Os, ClassCXXRD);
     Os << " is a "
-       << (isa<PointerType>(MemberType) ? "raw pointer" : "reference")
-       << " to ref-countable type ";
+       << (isa<PointerType>(MemberType) ? "raw pointer" : "reference") << " to "
+       << typeName() << " ";
     printQuotedQualifiedName(Os, MemberCXXRD);
```

Comment:
Have you tried a simple `Os << MemberCXXRD` here? I think it even adds quotes automatically when you do that. But I don't remember if it works for arbitrary streams or only for clang warnings streams, so this might be completely misleading.

Reply:
Hm... looks like `Os << MemberCXXRD` would output MemberCXXRD as a pointer value.